lib/show_for/builder.rb

```diff
@@ -84,6 +84,11 @@ def collection_block?(block) #:nodoc:
     def skip_blanks?(value) #:nodoc:
       ShowFor.skip_blanks && value.blank? && value != false
     end
+
+    def is_empty?(value)
```

Comment:
small thing. This method needs a `#:nodoc:` after the `)`

Reply:
Done!